server/monitor.go

```diff
@@ -752,6 +752,8 @@ func (s *Server) HandleRoutez(w http.ResponseWriter, r *http.Request) {
 
 // Subsz represents detail information on current connections.
 type Subsz struct {
+	ID  string    `json:"server_id"`
+	Now time.Time `json:"now"`
```

Comment:
sorry a bit late to the party here - but as we use `timestamp` elsewhere for the typed events can we use the same here given its a new field?

Reply:
Even though this is an addition, all of the other monitoring endpoints use `now` so I would prefer to keep that for consistency with the HTTP endpoints.

Reply:
@ripienaar I kept it the same way to be identical with the rest. If we expose the monitoring endpoints via nats, these are wrapped and the outer json will adhere to what we discussed.